api_tests/nodes/filters/test_filters.py

```diff
@@ -32,7 +32,7 @@ def setUp(self):
 
     def test_parent_filter_null(self):
         expected = [self.node_A._id]
-        res = self.app.get('{}null'.format(self.parent_url), auth=self.user.auth)
+        res = self.app.get('{}null'.format(self.parent_url), auth=self.user.auth, expect_errors=True)
```

Comment:
This test is confusing; expect_errors is `True`, but we assert that the returned data are valid?